Library/ViewModels/DiscoveryPostcardViewModel.swift

```diff
@@ -133,10 +150,20 @@ public protocol DiscoveryPostcardViewModelType {
 public final class DiscoveryPostcardViewModel: DiscoveryPostcardViewModelType,
   DiscoveryPostcardViewModelInputs, DiscoveryPostcardViewModelOutputs {
 
-    public init() {
-    let project = self.projectProperty.signal.skipNil()
+  // swiftlint:disable function_body_length
+  public init() {
+    let configuredProject = self.projectProperty.signal.skipNil()
+      .map(cached(project:))
+
+    let currentUser = Signal.merge([
+      self.userSessionStartedProperty.signal,
+      self.userSessionEndedProperty.signal,
+      configuredProject.ignoreValues()
```

Comment:
Yeah @christopherwright, as Christella is saying, the `currentUser` is referenced later when a logged out user tries to save a project. The reason to put it up here is just for the convenience of getting the current user in one place and filtering on it later when the button is tapped.